test/lib/unit/utils/lb_spec.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+const assert = require("assert");
+const sinon = require("sinon");
+const path = require("path");
+const LbHelper = require("../../../../lib/utils/lb");
+const LabShareConfig = require("../../../../lib/utils/labshare-config");
+const testModulePath = path.join(__dirname ,"../../../fixtures/package")
+
+
+describe('lb Utils', () => {
+
+    afterEach(() => {
+            sinon.restore();
+          
+        });
+
+    it('lb getApiRestSettings with defined config', async () => {
+     let lbHelper = new LbHelper();
+     let settings = lbHelper.getApiRestSettings({ "test":{test:1}} , "test");
+     expect(settings).toBeDefined()
+    });
+
+    it('lb getApiRestSettings with no config', async (done) => {
+      let lbHelper = new LbHelper();
+      try{
+        lbHelper.getApiRestSettings({ "test":{test:1}} , "testApi");
```

Comment:
There's a Jasmine helper for expectations on exceptions. I think it's expect.toThrow() or similar.